README.md

```diff
@@ -19,34 +20,22 @@ Pull requests are welcome.
 
 ## Getting started
 
-Clone this repository:
-
-```bash
+```
 $ git clone https://github.com/aviggiano/redis-roaring.git
 $ cd redis-roaring/
+$ configure.sh
```

Comment:
Could you also add these steps into the automated setup script?

https://github.com/aviggiano/redis-roaring/blob/master/configure.sh